src/utils/index.js

```diff
@@ -1,6 +1,7 @@
 import accepts from 'attr-accept'
 
-export const supportMultiple = 'multiple' in document.createElement('input')
```

Comment:
Wouldn't  it be enough  to just change 

`document.createElement('input')`

 to

`React.createElement('input')`  ?

Reply:
No, a React element won't really tell you what the browser is capable of, it's just a plain object afterall. The result of `React.createElement('input')` will look like this:
```js
{
  $$typeof: Symbol(react.element)
  props: {}
  type: "input"
}
```

Reply:
Ok thanks for the reply